source/refined-github.ts

```diff
@@ -44,8 +44,8 @@ import './features/one-key-formatting';
 import './features/tab-to-indent';
 import './features/hide-navigation-hover-highlight';
 import './features/selection-in-new-tab';
-import './features/hide-comments-faster';
-import './features/edit-comments-faster';
+import './features/hide-comments-quick';
+import './features/quick-comments-edit';
```

Comment:
```suggestion
import './features/hide-comments-quickly';
import './features/quickly-edit-comments';
```

Reply:
The suggestion was to basically change the _action_ name into to a description. "do-this" would just become "deed", just like we _don't_ have `go-to-latest-tag` but `latest-tag-button`